data/assets/scene/solarsystem/sssb/c2019y4atlas.asset

```diff
@@ -26,10 +29,9 @@ local C2019Y4AtlasTrail = {
     },
     Color = { 0.533333, 0.850980, 0.996078 },
     EnableFade = false,
-    StartTime = "1950 JAN 1 0:00:00",
-    EndTime = "2100 JAN 1 00:00:00",
-    SampleInterval = 52595, -- About 14h 36m 35.5200s in seconds
-    TimeStampSubsampleFactor = 1
+    StartTime = StartTime,
+    EndTime = EndTime,
+    SampleInterval = openspace.time.duration(StartTime, EndTime) / 90000
```

Comment:
Why divide by 90000? What does it mean? It could be nice with a comment that describes it. It would be nice also to keep the old comment with how long in times this equals too as well.

Reply:
Line 28 just above has the "Step Size: 90000" line.  I wouldn't want to put in the exact number of dates since we use the `StartTime`, `EndTime` to make it more flexible and I think the comment would get out of date